dagql/idtui/symbols.go

```diff
@@ -0,0 +1,35 @@
+package idtui
+
+const (
+	Block               = "█"
+	CaretDownEmpty      = "▽"
+	CaretDownFilled     = "▼"
+	CaretLeftFilled     = "◀" //"<"
+	CaretRightEmpty     = "▷" //">"
+	CaretRightFilled    = "▶" //">"
```

Comment:
There is a tiny `golint` issue here: https://github.com/dagger/dagger/actions/runs/9514121731/job/26225628274#step:3:441

<img width="1179" alt="image" src="https://github.com/dagger/dagger/assets/3342/bb6f186a-8f61-464e-9047-796437e1e09a">

Reply:
Already have an open PR: https://github.com/dagger/dagger/pull/7666